apps/heft/src/plugins/TypeScriptPlugin/Tslint.ts

```diff
@@ -36,28 +36,63 @@ export class Tslint extends LinterBase<TTslint.RuleFailure> {
     this._fileSystem = options.fileSystem;
   }
 
+  private static _resolvePackageMainFilePath(packagePath: string): string {
+    const packageJson: IPackageJson | undefined = this._packageJsonLookup.tryLoadPackageJsonFor(packagePath);
+    if (!packageJson) {
+      throw new Error(
+        `Supplied package path ${packagePath} could not resolve a package.json while searching for main file`
+      );
+    } else if (!packageJson.main) {
+      throw new Error(`Package at path ${packagePath} specifies no main file`);
+    }
+    return path.resolve(packagePath, packageJson.main);
```

Comment:
This isn't necessarily correct. You may need to `.realPath` the `packagePath`

Reply:
Okay, do you know what type that property belongs to?

Reply:
Ended up removing this function